From fc4c992573311e40f922a45cef0251bf26f1c894 Mon Sep 17 00:00:00 2001
From: Mauro Carvalho Chehab <mchehab@s-opensource.com>
Date: Wed, 26 Oct 2016 15:25:23 -0200
Subject: [PATCH 2885/2912] edac: edac_core.h: get rid of unused kobj_complete

This element of struct edac_pci_ctl_info is never used. So,
get rid of it.

Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
Signed-off-by: Sudheesh Mavila <sudheesh.mavila@amd.com>
---
 drivers/edac/edac_core.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/edac/edac_core.h b/drivers/edac/edac_core.h
index 4861542..58d66da 100644
--- a/drivers/edac/edac_core.h
+++ b/drivers/edac/edac_core.h
@@ -380,7 +380,6 @@ struct edac_pci_ctl_info {
 	 * device this structure controls
 	 */
 	struct kobject kobj;
-	struct completion kobj_complete;
 };
 
 #define to_edac_pci_ctl_work(w) \
-- 
2.7.4

